doc/rados/configuration/ceph-conf.rst

```diff
@@ -298,10 +314,13 @@ combined with the string in the next line. Here is an example::
   long ago
 
 In this example, the value of the "``foo``" option is "``long long ago long
-ago``".
+ago``".  Be careful to not place a backslash at the end of the final line
+of the multi-line string.
 
-An option value typically ends with either a newline or a comment. For
-example:
+An option value setting in a local config file ends with a newline.
```

Comment:
I think in a future PR, we should change the apparent noun string "option value setting" so that it has quotes around it (or something), so that it is clear that it is a compound noun.